audits/token/202312-threat-model-preset-erc1155.md

```diff
@@ -1,28 +1,41 @@
 ## Introduction
-This document is a thread model for the preset erc1155 token contracts built by Immutable.
 
-This document encompasses information for all contracts under the [token](../contracts/token/erc1155) directory
+This document is a thread model for the preset ERC1155 token contracts built by Immutable.
+
+Contracts covered under this model include:
+
+- [ImmutableERC1155](../../contracts/token/erc1155/preset/ImmutableERC1155.sol)
```

Comment:
I would have this link to a specific githash. Given you are using the diagram from the README, it is conceivable that the diagram may change.

Reply:
Have updated this link and also links for underlying contracts, to keep them all in sync: https://github.com/immutable/contracts/pull/229/commits/cef41759081a0a4a95587d0e4ccf4dab72e4777a